test/net/sourceforge/kolmafia/swingui/panel/DailyDeedsPanelTest.java

```diff
@@ -279,6 +280,7 @@ public void showsLocket3Monsters() {
   class DropsFamiliars {
     @Test
     public void showsCookbookbat() {
+      FamiliarData.reset();
```

Comment:
Put this into a BeforeEach method for this Nested class?

Reply:
Did not realize I could do that.  Done.  Thank you.